canonical-web-and-design/snapcraft.io
# Switch over to using canonicalwebteam.launchpad

The code from webapp/api/launchpad.py was ported to
canonicalwebteam.launchpad, so it could be shared by ubuntu.com's
image builder stuff. It just means a few small interface changes.

Update this code to use that new module and interface, and remove
the old code from this codebase.

QA
--

Run the site, go to your publisher section, find a snap, go to the "builds" tab.

Try adding a repo to build from, and try triggering a build. Check everything works as expected.
